brain/synapses.cpp

```diff
@@ -123,14 +139,18 @@ struct Synapses::Impl : public Synapses::BaseImpl
             return;
 
         const brion::Synapse& synapseAttributes =
-            _circuit._impl->getSynapseAttributes(_afferent);
-        const brion::Synapse* synapseExtra = _circuit._impl->getSynapseExtra();
+            _source.empty()
```

Comment:
rename _source to _externalSource or similar? It's not quite obvious deeply in the code what this means.